### PR TITLE
button component done

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
Completed button component.
please review it.

### CODE REVIEW

File: button/button.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    border: 1px solid var(--RED);
+}
+
+.btn--outline {
+    background-color: transparent;
+}
+
+.btn--round {
+    border-radius: 50px;
+    /* border: none; */
```
Comment (theVivek-kumar) [inline button/button.css]: code is very clean I like the naming

File: button/button.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+                    Link
+                </a>
+            </div>
+
+    </div>
+    </main>
+    </div>
+
+</body>
+
```
Comment (theVivek-kumar) [inline button/button.html:189]: you added the comments in between make code more readable

File: css/main.css [hunk 3]
```diff
@@ -27,8 +27,8 @@ html{
     --WHITE: #ffffff;
```
Comment (theVivek-kumar) [inline css/main.css:27]: suggestion make color name little btter

**Review by theVivek-kumar — APPROVED** [review]
looks fine to me you can mearge now

File: button/button.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    justify-content: space-around;
+    margin: 2rem;
+    max-width: 80%;
+    width: auto; 
+    border-radius: 0.2rem;
+}
+
+.btn {
+    padding: 1rem 2rem;
+    font-family: 'Montserrat', sans-serif;;
```
Comment (git-0r) [inline button/button.css]: 1. Extra semicolon on line 19.
2. If you are re-using .btn class, then why are you adding blue border & background ?
3. You can add border-width & style to btn class then apply border-color to particular button.

File: button/button.html [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+        <!---------------------  main-content -------------------------->
+
+        <main class="main--content">
+
+            <h2 class="intoduction--heading">Buttons</h2>
+            <p class="introduction--para">Buttons are the soul of an interactive Web Application. Buttons allow users to
+                take actions, and make choices, with a single tap.
+            </p>
+            <div class="divider"></div>
```
Comment (git-0r) [inline button/button.html:136]: you can use <strong> hr </strong> instead of div with divider class.
<hr>

File: button/button.css [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+.btn--round {
+    border-radius: 50px;
+    /* border: none; */
+}
+
+
+.btn--outline:hover {
+    background-color: inherit;
+    color: var(--WHITE);
+}
```
Comment (nikhiljugale007) [inline button/button.css:101]: on hove you can add some more style to look it better, currently, I guess just curser I changing to the pointer.

File: button/button.css [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+.btn--ghost {
+    background-color: var(--GRAY-80);
+    border: none;
+    color: var(--WHITE);
+    font-size: 20px;
+    cursor: pointer;
```
Comment (nikhiljugale007) [inline button/button.css]: you can use curser: no-drop for disabled button

**Review by nikhiljugale007 — COMMENTED** [review]
Overall UI and code are good.
Added a few suggestions in the code.

File: button/button.css (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply button/button.css]: Thanks,
I've used BEM convention here.

File: button/button.css (hunk 4 above)
Comment (Adarsh-Patel-Dev) [reply button/button.css]: Thanks for these suggestions
I'll change accordingly

File: button/button.css (hunk 7 above)
Comment (Adarsh-Patel-Dev) [reply button/button.css]: Thanks I was not aware about this property

File: button/button.css (hunk 6 above)
Comment (Adarsh-Patel-Dev) [reply button/button.css:101]: Done with hover effect

File: button/button.html (hunk 5 above)
Comment (Adarsh-Patel-Dev) [reply button/button.html:136]: Good suggestion
but i need to change it in every component then🙂

**Pull request merged**